spec/griddler/sendgrid/adapter_spec.rb

```diff
@@ -124,10 +128,20 @@
     normalized_params[:bcc].should eq []
   end
 
+  it 'distinguishes between angle brackets in the display name and the email address' do
+    params = default_params.merge(
+      to: '"<Display name>" <email@example.com>',
+      envelope: '{"to":["email@example.com"]}',
+    )
+    normalized_params = normalize_params(params)
+
+    normalized_params[:bcc].should eq []
```

Comment:
@sdhull It was a bit hard to explain what was going on here, but this test verifies two behaviors in conjunction:

- Returns an empty array when the envelope `to` is the same as the base `to` (copied from the test above this one)
- Angle brackets in the name portion are not parsed as email addresses

These two behaviors are verified by ensuring that the `to` defined here is parsed as `email@example.com`, which matches the envelope `to` and thus returns an empty `bcc`.

In retrospect, correctly ignoring non-email angle brackets can be demonstrated separate from the `bcc` behavior tested above. Something like this could work:
```ruby
params = default_params.merge(
  to: '"<display-name@example.com>" <email@example.com>, "<Display name>" <another-email@example.com>',
)
normalized_params = normalize_params(params)

normalized_params[:to].should eq [
  '"<display-name@example.com>" <email@example.com>',
  '"<Display name>" <another-email@example.com>',
]
```
Thoughts?